test/unit/application_load_paths_test.rb

```diff
@@ -0,0 +1,50 @@
+# typed: false
+# frozen_string_literal: true
+
+require "test_helper"
+require "rails_test_helper"
+
+module Packwerk
+  class ApplicationLoadPathsTest < Minitest::Test
```

Comment:
Should we add a test case for paths that do not exist? Which is the issue we are seeing in https://github.com/Shopify/packwerk/pull/65

Reply:
Definitely! I had actually thought about that at some point and then forgot about it 🤦

Reply:
Thank you Maple!

Reply:
Mmmh, revisiting this - I don't think non-existent paths were the problem; the problem was that the paths were outside the project directory. Which is tested here.

Reply:
I should however add a test for a more complex path, like `/application/../something/`.

Reply:
I can't / don't want to use `.realpath` because that accesses the file system, which makes testing hard.

Reply:
> Mmmh, revisiting this - I don't think non-existent paths were the problem; the problem was that the paths were outside the project directory. Which is tested here.

Yes, sorry for the confusion, that was what I meant

Reply:
Alignment!